libs/gamify/data/src/lib/services/applications.service.ts

```diff
@@ -12,8 +13,17 @@ export class ApplicationsService {
         private readonly usersService: UsersService
     ) {}
 
-    async findMany(): Promise<Application[]> {
-        return (await this.data.application.findMany()).map(app => ApplicationConverter.fromPrismaApplication(app));
+    async findMany(filter?: string|undefined, user?: UserModel): Promise<Application[]> {
```

Comment:
'filter' as a string type, dedicated to filter only on 'moderatable': in this case the name filter seems to be too generic. You could give this parameter a more specific name, or implement some 'object' or 'dictionary' datatype so that this parameter could hold more filters in the future. 

But for now no action needed.